test/core/conversions.wast

```diff
@@ -120,6 +120,8 @@
 (assert_return (invoke "i32.trunc_f64_s" (f64.const -2.0)) (i32.const -2))
 (assert_return (invoke "i32.trunc_f64_s" (f64.const 2147483647.0)) (i32.const 2147483647))
 (assert_return (invoke "i32.trunc_f64_s" (f64.const -2147483648.0)) (i32.const -2147483648))
+(assert_return (invoke "i32.trunc_f64_s" (f64.const 2147483647.9999998)) (i32.const 2147483647))
+(assert_trap   (invoke "i32.trunc_f64_s" (f64.const 2147483647.9999999)) "integer overflow")
```

Comment:
This value seems to be equivalent to `2147483648.0`, so it's really just testing the wat double parsing; I'm assuming we don't care much about that here.

Reply:
Yeah, to be fair I think we've done this for a few other tests too, so perhaps we should leave it as is here too. Personally, I'd prefer to use hex floats everywhere that we're not explicitly testing text parsing, but that should be done as a different change.